lib/TraackrApi/TraackrApiObject.php

```diff
@@ -256,7 +256,7 @@ public function post($url, $params = [])
         // Build Parameters
         // Add API key parameter if not present
         $api_key = TraackrApi::getApiKey();
-        if (!isset($params[PARAM_API_KEY]) && !empty($api_key)) {
+        if (!isset($params[PARAM_API_KEY]) && !empty($api_key) && !$isJson) {
```

Comment:
This `!$isJson` check seems like a kind of roundabout way of inferring whether the API key should be included. Would it be better to add a `$apiKeyRequired = true` param to the post method that we could set to false in the case of `keywords`?

Reply:
API key is only fetched from the query_String in the back-end , so no need to put it in the BODY

Reply:
@jec3 @bbguitar77 I've removed the api-key from the body of POST and DELETE requests.